react-form-controls/src/NewsletterForm.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {email: ""};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleChange = (event) => {
+    this.setState({email: event.target.value})
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault()
+    console.log(this.state)
+  }
+
+  render(){
+    return (
+      <form className = "flex">
+        <span>Email</span>
+        <input type="text"
+          onChange={this.handleChange}
+          />
+        <button type="button"
+          onSubmit={this.handleSubmit}
```

Comment:
`handleSubmit` should be on the `form` elements `onSubmit` prop, not on the `button`

Reply:
The `onSubmit` should be on the opening `<form>` tag on line 21